src/utils/ServerInterfaceStaff.tsx

```diff
@@ -0,0 +1,165 @@
+// Server-client interface for /api/users/staff
+import { staffType } from "./models/staffModel";
+
+const BACKEND_ROUTE = `${import.meta.env.VITE_BACKEND_ROUTE}/users/staff`;
+
+// JSON response checker - TODO: change to use jack's responseHelpers.tsx once merged   
+async function checkJSONResponseStatus(response: Response) {
+    if(!response.ok) {
+        throw new Error(response.statusText);
+    }
+    let parsedResponse = await response.json();
+    return parsedResponse;
+}
+
+// Miscellaneous response checker - TODO: change to use jack's responseHelpers.tsx once merged 
+async function checkMiscResponseStatus(response : Response) {
+    if(!response.ok) {
+        throw new Error(response.statusText);
+    }
+    return true;
+}
+
+// GET all staff
```

Comment:
A lot of these comments are made redundant by the function name. Only write comments when it adds value to the code! Every line of code is tech debt :)